pkgs/development/libraries/science/math/mkl/test/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkg-config, mkl }:
+
+stdenv.mkDerivation {
+  pname = "mkl-test";
+  version = mkl.version;
+
+  src = ./.;
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ mkl ];
+
+  doCheck = true;
+
+  buildPhase = ''
+    # Check regular Nix build.
+    gcc $(pkg-config --cflags --libs mkl-dynamic-ilp64-seq) test.c -o test
+
+    # Clear flags to ensure that we are purely relying on options
+    # provided by pkg-config.
+    NIX_CFLAGS_COMPILE="" \
+    NIX_LDFLAGS="" \
+      gcc $(pkg-config --cflags --libs mkl-dynamic-ilp64-seq) test.c -o test
```

Comment:
```suggestion
    NIX_LDFLAGS="" \
    gcc $(pkg-config --cflags --libs mkl-dynamic-ilp64-seq) test.c -o test
```

Reply:
I don't know, I like the original formatting better, it sets apart the variables from the command.

Reply:
It is inconsistent with the line above. Either intend the last 2 lines or intent nothing.